### ISSUE TITLE
Custom cowsay executable

### ISSUE THREAD

**Issue by clonedagain** [ansible/ansible#18879]
##### ISSUE TYPE
 - Feature Idea

##### COMPONENT NAME
Cowsay support.
And... ponies!

##### ANSIBLE VERSION
```
ansible 2.3.0 (devel f1b37a184c) last updated 2016/12/07 17:17:58 (GMT +200)
  config file = 
  configured module search path = Default w/o overrides
```

##### CONFIGURATION
None

##### OS / ENVIRONMENT
N/A

##### SUMMARY
I'd like to use `ponysay` (https://github.com/erkin/ponysay) instead of `cowsay`.

Others would like to use `cowsay` from a specific location for reasons like building cowsay without root privilege or using homebrew (https://github.com/ansible/ansible/issues/13465).

An option to set the cowsay path instead of guessing it would be useful to us minority-cowsay-variants enthusiasts.

It would be a nice complement if https://github.com/ansible/ansible/issues/18873 is fixed too.

##### STEPS TO REPRODUCE
Setting `ANSIBLE_COWSAY_EXECUTABLE` would be a nice way to explicitly set the path to cowsay.
The executable itself would not need to be named `cowsay`.

##### EXPECTED RESULTS
Colorful ponies when using `ANSIBLE_COWSAY_EXECUTABLE=/usr/bin/ponysay`.

##### ACTUAL RESULTS
As a workaround we currently need to link our executables to one of the paths supported by Ansible.

**Comment by jctanner** [comment]
needs_contributor

**Comment by clonedagain** [comment]
I was delighted to notice this was fixed in v2.5.0 : https://github.com/ansible/ansible/commit/95ff8f1a90f9008dd8ae91df4655b4cc39cb210e

I can use Ponysay by setting `ANSIBLE_COW_PATH=$(which ponysay)`

Please close this issue, and everypony celebrate !

**Comment by bcoca** [comment]
closing as per above
